README.md
<!-- copyright 2020 yearn.finance && the contributors -->
<p align="center">
 <img src="https://raw.githubusercontent.com/gist/sambacha/9a62174a080ead34271ff5d187a24343/raw/02cad2900f4cedb87895cc57729030e095d2869f/gh_banner.svg" align="right" width="350">
	<h1 align="left">Yearn Finance Ecosystem Tokenlist </h1>
 <h3 align="center"> </h3>
 <p align="center">
<align="center">

[![Twitter Follow](https://img.shields.io/twitter/follow/iearnfinance.svg?label=iearnfinance&style=social)](https://twitter.com/iearnfinance) [![Discord](https://img.shields.io/discord/734804446353031319.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discordapp.com/channels/734804446353031319/) [![Telegram](https://img.shields.io/badge/chat-on%20Telegram-blue.svg)](https://t.me/yearnfinance) <img alt="Discourse users" src="https://img.shields.io/discourse/users?server=https%3A%2F%2Fgov.yearn.finance%2F">

 </center>
  </p>
</p>
<br />
<br />

## Resources

- [Yearn Finance](https://yearn.finance)
- [Ethereum token lists](https://tokenlists.org/)
- [https://github.com/Uniswap/token-lists](https://github.com/Uniswap/token-lists)
- [Semantic versioning](https://github.com/Uniswap/token-lists/blob/master/README.md#semantic-versioning)

## Overview

```bash
npm install yearn-tokenlist
```

[npm yearn-tokenlist](https://www.npmjs.com/package/yearn-tokenlist)

<br>

### Local Testing

The `tokens.json` is validated against Uniswap's tokenlist schema.

To test locally,

```
# Install dependencies
npm install

# Run test script
npm start
```

If there is an error, the error will be output to the terminal.

## Deployed Contracts Registry

Below is a list of relevant smart contracts and Github repositories for using and interacting with the Yearn product suite.

## Token Contracts

### YFI

The Yearn ecosystem is controlled by YFI token holders who submit and vote on proposals that govern the ecosystem.

| Token                                                                        | Address                                    |
| ---------------------------------------------------------------------------- | ------------------------------------------ |
| [YFI](https://etherscan.io/token/0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e) | 0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e |

### v2 Yield Tokens

The v2 yield tokens can be used in one lender at a time. Currently being used on Curve’s Y Pool, Yearn’s yBTC pool and sUSD pool.

| Token                                                                              | Address                                    | Github                                                                           |
| ---------------------------------------------------------------------------------- | ------------------------------------------ | -------------------------------------------------------------------------------- |
| [yDAIv2](https://etherscan.io/address/0x16de59092dAE5CcF4A1E6439D611fd0653f0Bd01)  | 0x16de59092dAE5CcF4A1E6439D611fd0653f0Bd01 | [YDAIv2.sol](https://github.com/yearn/itoken/blob/master/contracts/YDAIv2.sol)   |
| [yUSDCv2](https://etherscan.io/address/0xd6aD7a6750A7593E092a9B218d66C0A814a3436e) | 0xd6aD7a6750A7593E092a9B218d66C0A814a3436e | [YUSDCv2.sol](https://github.com/yearn/itoken/blob/master/contracts/YUSDCv2.sol) |
| [yUSDTv2](https://etherscan.io/address/0x83f798e925BcD4017Eb265844FDDAbb448f1707D) | 0x83f798e925BcD4017Eb265844FDDAbb448f1707D | [YUSDTv2.sol](https://github.com/yearn/itoken/blob/master/contracts/YUSDTv2.sol) |
| [ysUSDv2](https://etherscan.io/address/0xF61718057901F84C4eEC4339EF8f0D86D2B45600) | 0xF61718057901F84C4eEC4339EF8f0D86D2B45600 | [YSUSDv2.sol](https://github.com/yearn/itoken/blob/master/contracts/YSUSDv2.sol) |
| [yTUSDv2](https://etherscan.io/address/0x73a052500105205d34daf004eab301916da8190f) | 0x73a052500105205d34daf004eab301916da8190f | [YTUSDv2.sol](https://github.com/yearn/itoken/blob/master/contracts/YTUSDv2.sol) |
| [yWBTCv2](https://etherscan.io/address/0x04Aa51bbcB46541455cCF1B8bef2ebc5d3787EC9) | 0x04Aa51bbcB46541455cCF1B8bef2ebc5d3787EC9 | [YWBTCv2.sol](https://github.com/yearn/itoken/blob/master/contracts/YWBTCv2.sol) |

### v3 Yield Tokens

The v3 yield tokens allow the underlying assets to be spread across multiple lenders. Currently being used on Curve’s BUSD pool.

| Token                                                                              | Address                                    | Github                                                                           |
| ---------------------------------------------------------------------------------- | ------------------------------------------ | -------------------------------------------------------------------------------- |
| [yDAIv3](https://etherscan.io/address/0xC2cB1040220768554cf699b0d863A3cd4324ce32)  | 0xC2cB1040220768554cf699b0d863A3cd4324ce32 | [YDAIv3.sol](https://github.com/yearn/itoken/blob/master/contracts/YDAIv3.sol)   |
| [yUSDCv3](https://etherscan.io/address/0x26EA744E5B887E5205727f55dFBE8685e3b21951) | 0x26EA744E5B887E5205727f55dFBE8685e3b21951 | [YUSDCv3.sol](https://github.com/yearn/itoken/blob/master/contracts/YUSDCv3.sol) |
| [yUSDTv3](https://etherscan.io/address/0xE6354ed5bC4b393a5Aad09f21c46E101e692d447) | 0xE6354ed5bC4b393a5Aad09f21c46E101e692d447 | [YUSDCv3.sol](https://github.com/yearn/itoken/blob/master/contracts/YUSDCv3.sol) |
| [yBUSDv3](https://etherscan.io/address/0x04bC0Ab673d88aE9dbC9DA2380cB6B79C4BCa9aE) | 0x04bC0Ab673d88aE9dbC9DA2380cB6B79C4BCa9aE | [YBUSDv3.sol](https://github.com/yearn/itoken/blob/master/contracts/YBUSDv3.sol) |

## Vault Contracts

Vaults follow unique strategies that are designed to maximize yield of the deposited asset and minimize risk. The vaults are maintained by a `Controller` which oversees the strategy execution. Strategies' generated profits are used to buy respective vault underlying asset.

The `Controller` contract can be found [here](https://etherscan.io/address/0x9e65ad11b299ca0abefc2799ddb6314ef2d91080#code) and the source code can be found [here](https://github.com/yearn/yearn-protocol/blob/develop/contracts/controllers/Controller.sol).

### Vaults

| Vault                                                                                             | Address                                    | GitHub                                                                                         |
| ------------------------------------------------------------------------------------------------- | ------------------------------------------ | ---------------------------------------------------------------------------------------------- |
| [WETH](https://etherscan.io/address/0xe1237aa7f535b0cc33fd973d66cbf830354d16c7)                   | 0xe1237aa7f535b0cc33fd973d66cbf830354d16c7 | [yWETH.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yWETH.sol)   |
| [YFI](https://etherscan.io/address/0xba2e7fed597fd0e3e70f5130bcdbbfe06bb94fe1)                    | 0xba2e7fed597fd0e3e70f5130bcdbbfe06bb94fe1 | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [curve.fi/3pool LP](https://etherscan.io/address/0x9cA85572E6A3EbF24dEDd195623F188735A5179f#code) | 0x9cA85572E6A3EbF24dEDd195623F188735A5179f | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [curve.fi/y LP](https://etherscan.io/address/0x5dbcf33d8c2e976c6b560249878e6f1491bca25c)          | 0x5dbcf33d8c2e976c6b560249878e6f1491bca25c | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [curve.fi/busd LP](https://etherscan.io/address/0x2994529c0652d127b7842094103715ec5299bbed)       | 0x2994529c0652d127b7842094103715ec5299bbed | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [curve.fi/sbtc LP](https://etherscan.io/address/0x7ff566e1d69deff32a7b244ae7276b9f90e9d0f6)       | 0x7ff566e1d69deff32a7b244ae7276b9f90e9d0f6 | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [DAI](https://etherscan.io/address/0xacd43e627e64355f1861cec6d3a6688b31a6f952)                    | 0xacd43e627e64355f1861cec6d3a6688b31a6f952 | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [TUSD](https://etherscan.io/address/0x37d19d1c4e1fa9dc47bd1ea12f742a0887eda74a)                   | 0x37d19d1c4e1fa9dc47bd1ea12f742a0887eda74a | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [USDC](https://etherscan.io/address/0x597ad1e0c13bfe8025993d9e79c69e1c0233522e)                   | 0x597ad1e0c13bfe8025993d9e79c69e1c0233522e | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [USDT](https://etherscan.io/address/0x2f08119c6f07c006695e079aafc638b8789faf18)                   | 0x2f08119c6f07c006695e079aafc638b8789faf18 | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [GUSD](https://etherscan.io/address/0xec0d8d3ed5477106c6d4ea27d90a60e594693c90)                   | 0xec0d8d3ed5477106c6d4ea27d90a60e594693c90 | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [curve.fi/compound LP](https://etherscan.io/address/0x629c759d1e83efbf63d84eb3868b564d9521c129)   | 0x629c759D1E83eFbF63d84eb3868B564d9521C129 | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |
| [curve.fi/mUSD LP](https://etherscan.io/address/0x1AEf73d49Dedc4b1778d0706583995958Dc862e6)       | 0x1AEf73d49Dedc4b1778d0706583995958Dc862e6 | [yVault.sol](https://github.com/yearn/yearn-protocol/blob/develop/contracts/vaults/yVault.sol) |

### Strategies

| Strategy                                                                                                   | Address                                    | GitHub                                                                                                                                               | Vault                                                                                           |
| ---------------------------------------------------------------------------------------------------------- | ------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |

CHANGELOG.md
### Changelog

All notable changes to this project will be documented in this file. Dates are displayed in UTC.

Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).

#### [v1.1.5](https://github.com/sambacha/yearn-finance-tokenlist/compare/v1.1.2...v1.1.5)

> 26 March 2021

- Tokenlists 2021.03.26 [`#34`](https://github.com/sambacha/yearn-finance-tokenlist/pull/34)
- chore(sync): sushiswap: [`#33`](https://github.com/sambacha/yearn-finance-tokenlist/pull/33)
- chore(deps): update dependency @types/fs-extra to v9.0.9 [`#31`](https://github.com/sambacha/yearn-finance-tokenlist/pull/31)
- chore(deps): update dependency ajv to v6.12.6 [`#20`](https://github.com/sambacha/yearn-finance-tokenlist/pull/20)
- v1.0.3 [`#27`](https://github.com/sambacha/yearn-finance-tokenlist/pull/27)
- refactor(lint): rebase [`6b57c51`](https://github.com/sambacha/yearn-finance-tokenlist/commit/6b57c5135f7ef1a18afbd3334b9a05120b42412d)
- chore(repo): cleanup [`56633f1`](https://github.com/sambacha/yearn-finance-tokenlist/commit/56633f149f3ac9f411b1afc763fe695b52eda969)
- feat(v1.1.5): release [`0fa1829`](https://github.com/sambacha/yearn-finance-tokenlist/commit/0fa182987a51418de583a6795c409f20353d013d)

#### [v1.1.2](https://github.com/sambacha/yearn-finance-tokenlist/compare/v1.1.1...v1.1.2)

> 18 March 2021

- feat(kodiak): automerge [`#26`](https://github.com/sambacha/yearn-finance-tokenlist/pull/26)
- chore(sync): sushiswap: [`#22`](https://github.com/sambacha/yearn-finance-tokenlist/pull/22)
- chore(sync): sushiswap: [`#21`](https://github.com/sambacha/yearn-finance-tokenlist/pull/21)
- chore(deps): pin dependencies [`#19`](https://github.com/sambacha/yearn-finance-tokenlist/pull/19)
- feat: API [`#18`](https://github.com/sambacha/yearn-finance-tokenlist/pull/18)
- chore(sync): sushiswap: [`#17`](https://github.com/sambacha/yearn-finance-tokenlist/pull/17)
- chore(deps): update dependency semantic-release to v17.4.1 [`#14`](https://github.com/sambacha/yearn-finance-tokenlist/pull/14)

#### [v1.1.1](https://github.com/sambacha/yearn-finance-tokenlist/compare/v1.1.0...v1.1.1)

> 5 March 2021

- chore(sync): sushiswap: Sat Mar  6 00:55:20 UTC 2021 [`#15`](https://github.com/sambacha/yearn-finance-tokenlist/pull/15)
- action(fix): set env $d [`f6d2c1b`](https://github.com/sambacha/yearn-finance-tokenlist/commit/f6d2c1b5244090bef4657602519acb13106e3b60)

#### [v1.1.0](https://github.com/sambacha/yearn-finance-tokenlist/compare/1.1.0...v1.1.0)

> 2 March 2021

- docs(chore): changelog [`212bf4e`](https://github.com/sambacha/yearn-finance-tokenlist/commit/212bf4ef3797e06aadb3bfb4b76ed3de87632237)
- feat(npm): v1.1.0 [`9ae190c`](https://github.com/sambacha/yearn-finance-tokenlist/commit/9ae190c2977a5ab6344957872d8f43a38389bb33)

#### [1.1.0](https://github.com/sambacha/yearn-finance-tokenlist/compare/1.0.6...1.1.0)

> 2 March 2021

- chore(sync): sushiswap: Tue Mar  2 00:45:57 UTC 2021 [`#13`](https://github.com/sambacha/yearn-finance-tokenlist/pull/13)
- chore(deps): update dependency semantic-release to v17.4.0 [`#12`](https://github.com/sambacha/yearn-finance-tokenlist/pull/12)
- chore(sync): sushiswap: Thu Feb 25 00:53:44 UTC 2021 [`#11`](https://github.com/sambacha/yearn-finance-tokenlist/pull/11)
- fix(git): -B flag [`#10`](https://github.com/sambacha/yearn-finance-tokenlist/pull/10)
- fix(lint): prettier config [`c294f94`](https://github.com/sambacha/yearn-finance-tokenlist/commit/c294f94b175d477b2dc294527649d06c618b050d)
- sync [`6d0bc58`](https://github.com/sambacha/yearn-finance-tokenlist/commit/6d0bc58b5f8adbc1c31f5062f5bad68e13cfe049)
- build(kodiak): init [`fc8d075`](https://github.com/sambacha/yearn-finance-tokenlist/commit/fc8d075d6ed1a1e02b4f57f6decdfe16846748e2)

#### [1.0.6](https://github.com/sambacha/yearn-finance-tokenlist/compare/1.0.5...1.0.6)

> 17 February 2021

- chore(sync): sushiswap: Wed Feb 17 15:54:53 UTC 2021 [`#8`](https://github.com/sambacha/yearn-finance-tokenlist/pull/8)
- Actions Pipeline [`#7`](https://github.com/sambacha/yearn-finance-tokenlist/pull/7)
- chore(deps): update dependency semantic-release to v17.3.9 [`#6`](https://github.com/sambacha/yearn-finance-tokenlist/pull/6)
- chore(deps): update dependency semantic-release to v17.3.7 [`#4`](https://github.com/sambacha/yearn-finance-tokenlist/pull/4)
- chore(deps): pin dependencies [`#3`](https://github.com/sambacha/yearn-finance-tokenlist/pull/3)
- chore(sync): sushiswap [`bce68e5`](https://github.com/sambacha/yearn-finance-tokenlist/commit/bce68e52b10820bcf930bb3d15bc62f417812a37)
- update(sushiswap): list [`91e5b37`](https://github.com/sambacha/yearn-finance-tokenlist/commit/91e5b37fd0fa707aa7afb830b55ca881f4635118)
- docs(readme): registry [`f355161`](https://github.com/sambacha/yearn-finance-tokenlist/commit/f355161c7af5c388b747ae51c16eabd3b1d2bd12)

#### [1.0.5](https://github.com/sambacha/yearn-finance-tokenlist/compare/1.0.4...1.0.5)

> 20 January 2021

- fix(lint): json defect [`8bcb8f5`](https://github.com/sambacha/yearn-finance-tokenlist/commit/8bcb8f515cca822a05afc5f1a743327452f57d99)
- Release 1.0.4 [`2aec2b8`](https://github.com/sambacha/yearn-finance-tokenlist/commit/2aec2b852e71aef379945ed3882b20a3a1325746)

#### [1.0.4](https://github.com/sambacha/yearn-finance-tokenlist/compare/1.0.3...1.0.4)

> 20 January 2021


index.js
module.exports = {
    "name": "Yearn Finance",
    "timestamp": "2021-01-20T06:23:16.000+00:00",
    "version": {
        "major": 1,
        "minor": 0,
        "patch": 0
    },
    "logoURI": "https://cdn.furucombo.app/assets/img/token/yYFI.png",
    "keywords": [
        "nomichef",
        "sushiswap",
        "cream",
        "cover",
        "keep3r",
        "yfi",
        "default",
        "list",
        "yearn finance",
        "interface",
        "aggregator",
        "vauults"
    ],
    "tokens": [{
            "name": "Curve Compound Pool yVault",
            "symbol": "yvCurve-Compound",
            "address": "0x629c759D1E83eFbF63d84eb3868B564d9521C129"
        },
        {
            "name": "Curve Compound Pool",
            "symbol": "crvCOMP",
            "address": "0x845838DF265Dcd2c412A1Dc9e959c7d08537f8a2"
        },
        {
            "name": "Curve 3pool yVault",
            "symbol": "yvCurve-3pool",
            "address": "0x9cA85572E6A3EbF24dEDd195623F188735A5179f"
        },
        {
            "name": "Curve 3pool",
            "symbol": "3Crv",
            "address": "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"
        },
        {
            "name": "Curve Y Pool yVault",
            "symbol": "yUSD",
            "address": "0x5dbcF33D8c2E976c6b560249878e6F1491Bca25c"
        },
        {
            "name": "Curve Y Pool",
            "symbol": "yCRV",
            "address": "0xdF5e0e81Dff6FAF3A7e52BA697820c5e32D806A8"
        },
        {
            "name": "Curve BUSD Pool yVault",
            "symbol": "yvCurve-BUSD",
            "address": "0x2994529C0652D127b7842094103715ec5299bBed"
        },
        {
            "name": "Curve BUSD Pool",
            "symbol": "crvBUSD",
            "address": "0x3B3Ac5386837Dc563660FB6a0937DFAa5924333B"
        },
        {
            "name": "Curve sBTC Pool yVault",
            "symbol": "yvCurve-sBTC",
            "address": "0x7Ff566E1d69DEfF32a7b244aE7276b9f90e9D0f6"
        },
        {
            "name": "Curve sBTC Pool",
            "symbol": "crvSBTC",
            "address": "0x075b1bb99792c9E1041bA13afEf80C91a1e70fB3"
        },
        {
            "name": "Curve GUSD Pool yVault",
            "symbol": "yvCurve-GUSD",
            "address": "0xcC7E70A958917cCe67B4B87a8C30E6297451aE98"
        },
        {
            "name": "Curve GUSD Pool",
            "symbol": "crvGUSD",
            "address": "0xD2967f45c4f384DEEa880F807Be904762a3DeA07"
        },
        {
            "name": "Curve mUSD Pool yVault",
            "symbol": "yvCurve-mUSD",
            "address": "0x0FCDAeDFb8A7DfDa2e9838564c5A1665d856AFDF"
        },
        {
            "name": "Curve mUSD Pool",
            "symbol": "crvMUSD",
            "address": "0x1AEf73d49Dedc4b1778d0706583995958Dc862e6"
        },
        {
            "name": "Curve stETH Pool yVault",
            "symbol": "yvCurve-stETH",
            "address": "0xdCD90C7f6324cfa40d7169ef80b12031770B4325"
        },
        {
            "name": "Curve stETH Pool",
            "symbol": "crvSTETH",
            "address": "0x06325440D014e39736583c165C2963BA99fAf14E"
        },
        {
            "name": "Curve EURS Pool yVault",
            "symbol": "yvCurve-EURS",
            "address": "0x98B058b2CBacF5E99bC7012DF757ea7CFEbd35BC"
        },
        {
            "name": "Curve EURS Pool",
            "symbol": "crvEURS",
            "address": "0x194eBd173F6cDacE046C53eACcE9B953F28411d1"
        },
        {
            "name": "Curve USDN Pool yVault",
            "symbol": "yvCurve-USDN",
            "address": "0xFe39Ce91437C76178665D64d7a2694B0f6f17fE3"
        },
        {
            "name": "Curve USDN Pool",
            "symbol": "crvUSDN",
            "address": "0x4f3E8F405CF5aFC05D68142F3783bDfE13811522"
        },
        {
            "name": "Curve UST Pool yVault",
            "symbol": "yvCurve-UST",
            "address": "0xF6C9E9AF314982A4b38366f4AbfAa00595C5A6fC"
        },
        {
            "name": "Curve UST Pool",
            "symbol": "crvUST",
            "address": "0x94e131324b6054c0D789b190b2dAC504e4361b53"
        },
        {
            "name": "Curve oBTC Pool yVault",
            "symbol": "yvCurve-oBTC",
            "address": "0x7F83935EcFe4729c4Ea592Ab2bC1A32588409797"
        },
        {
            "name": "Curve oBTC Pool",
            "symbol": "crvOBTC",
            "address": "0x2fE94ea3d5d4a175184081439753DE15AeF9d614"
        },
        {
            "name": "Curve pBTC Pool yVault",
            "symbol": "yvCurve-pBTC",
            "address": "0x123964EbE096A920dae00Fb795FFBfA0c9Ff4675"
        },
        {
            "name": "Curve pBTC Pool",
            "symbol": "crvPBTC",
            "address": "0xDE5331AC4B3630f94853Ff322B66407e0D6331E8"
        },
        {
            "name": "Curve tBTC Pool yVault",
            "symbol": "yvCurve-tBTC",
            "address": "0x07FB4756f67bD46B748b16119E802F1f880fb2CC"
        },
        {
            "name": "Curve tBTC Pool",
            "symbol": "crvTBTC",
            "address": "0x64eda51d3Ad40D56b9dFc5554E06F94e1Dd786Fd"
        },
        {
            "name": "Curve BBTC Pool yVault",
            "symbol": "yvCurve-BBTC",
            "address": "0xA8B1Cb4ed612ee179BDeA16CCa6Ba596321AE52D"
        },
        {
            "name": "Curve BBTC Pool",
            "symbol": "crvBBTC",
            "address": "0x410e3E86ef427e30B9235497143881f717d93c2A"
        },
        {
            "name": "Curve renBTC Pool yVault",
            "symbol": "yvCurve-renBTC",
            "address": "0x5334e150B938dd2b6bd040D9c4a03Cff0cED3765"
        },
        {
            "name": "Curve renBTC Pool",
            "symbol": "crvRENBTC",
            "address": "0x49849C98ae39Fff122806C06791Fa73784FB3675"
        },
        {
            "name": "DAI yVault",
            "symbol": "yvDAI",
            "address": "0xACd43E627e64355f1861cEC6d3a6688B31a6F952"
        },
        {
            "name": "DAI Stablecoin",
            "symbol": "DAI",
            "address": "0x6B175474E89094C44Da98b954EedeAC495271d0F"
        },
        {
            "name": "TUSD yVault",
            "symbol": "yvTUSD",
            "address": "0x37d19d1c4E1fa9DC47bD1eA12f742a0887eDa74a"
        },
        {
            "name": "TrueUSD",
            "symbol": "TUSD",
            "address": "0x0000000000085d4780B73119b644AE5ecd22b376"
        },
        {
            "name": "USDC yVault",
            "symbol": "yvUSDC",
            "address": "0x597aD1e0c13Bfe8025993D9e79C69E1c0233522e"
        },
        {
            "name": "USD Coin",
            "symbol": "USDC",
            "address": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
        },
        {
            "name": "USDT yVault",
            "symbol": "yvUSDT",
            "address": "0x2f08119C6f07c006695E079AAFc638b8789FAf18"
        },
        {
            "name": "Tether USD",
            "symbol": "USDT",
            "address": "0xdAC17F958D2ee523a2206206994597C13D831ec7"
        },
        {
            "name": "GUSD yVault",
            "symbol": "yvGUSD",
            "address": "0xec0d8D3ED5477106c6D4ea27D90a60e594693C90"
        },
        {
            "name": "Gemini Dollar",
            "symbol": "GUSD",
            "address": "0x056Fd409E1d7A124BD7017459dFEa2F387b6d5Cd"
        },
        {
            "name": "YFI yVault",
            "symbol": "yvYFI",
            "address": "0xBA2E7Fed597fd0E3e70f5130BcDbbFE06bB94fe1"
        },
        {
            "name": "yearn.finance",
            "symbol": "YFI",
            "address": "0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e"
        },
        {
            "name": "aLINK yVault",
            "symbol": "yvaLINK",
            "address": "0x29E240CFD7946BA20895a7a02eDb25C210f9f324"
        },
        {
            "name": "Aave Interest Bearing LINK",
            "symbol": "aLINK",
            "address": "0xA64BD6C70Cb9051F6A9ba1F163Fdc07E0DfB5F84"
        },
        {
            "name": "WETH yVault",
            "symbol": "yvWETH",
            "address": "0xe1237aA7f535b0CC33Fd973D66cBf830354D16c7"
        },
        {
            "name": "Wrapped Ether",
            "symbol": "WETH",
            "address": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
        },
        {
            "name": "Ether",
            "symbol": "ETH",
            "address": "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
        },
        {
            "name": "LINK yVault",
            "symbol": "yvLINK",
            "address": "0x881b06da56BB5675c54E4Ed311c21E54C5025298"
        },
        {
            "name": "ChainLink Token",
            "symbol": "LINK",
            "address": "0x514910771AF9Ca656af840dff83E8264EcF986CA"
        },
        {
            "name": "HEGIC yVault",
            "symbol": "yvHEGIC",
            "address": "0xe11ba472F74869176652C35D30dB89854b5ae84D"
        },
        {
            "name": "Hegic",
            "symbol": "HEGIC",
            "address": "0x584bC13c7D411c00c01A62e8019472dE68768430"
        },
        {
            "name": "SushiSwap YFI-ETH LP yVault",
            "symbol": "yvSushi-YFI-ETH",
            "address": "0x27Eb83254D900AB4F9b15d5652d913963FeC35e3"
        },
        {
            "name": "SushiSwap YFI-ETH LP",
            "symbol": "YFI-ETH SLP",
            "address": "0x088ee5007C98a9677165D78dD2109AE4a3D04d0C"
        },
        {
            "name": "DAI yVault",
            "symbol": "yvDAI",
            "address": "0x19D3364A399d251E894aC732651be8B0E4e85001"
        },
        {
            "name": "USDC yVault",
            "symbol": "yvUSDC",
            "address": "0x5f18C75AbDAe578b483E5F43f12a39cF75b973a9"
        },
        {
            "name": "mUSD yVault",
            "symbol": "yvmUSD",
            "address": "0xE0db48B4F71752C4bEf16De1DBD042B82976b8C7"
        },
        {
            "name": "mStable USD",
            "symbol": "mUSD",
            "address": "0xe2f2a5C287993345a840Db3B0845fbC70f5935a5"
        },
        {
            "name": "Curve DUSD Pool yVault",
            "symbol": "yvCurve-DUSD",
            "address": "0x8e6741b456a074F0Bc45B8b82A755d4aF7E965dF"
        },
        {
            "name": "Curve DUSD Pool",
            "symbol": "crvDUSD",
            "address": "0x3a664Ab939FD8482048609f652f9a0B0677337B9"
        },
        {
            "name": "Curve sUSD Pool yVault",
            "symbol": "yvCurve-sUSD",
            "address": "0x5533ed0a3b83F70c3c4a1f69Ef5546D3D4713E44"
        },
        {
            "name": "Curve sUSD Pool",
            "symbol": "crvSUSD",
            "address": "0xC25a3A3b969415c80451098fa907EC722572917F"
        },
        {
            "name": "Curve HBTC Pool yVault",
            "symbol": "yvCurve-HBTC",
            "address": "0x46AFc2dfBd1ea0c0760CAD8262A5838e803A37e5"
        },
        {
            "name": "Curve HBTC Pool",
            "symbol": "crvHBTC",
            "address": "0xb19059ebb43466C323583928285a49f558E572Fd"
        },
        {
            "name": "Curve HUSD Pool yVault",
            "symbol": "yvCurve-HUSD",
            "address": "0x39546945695DCb1c037C836925B355262f551f55"
        },
        {
            "name": "Curve HUSD Pool",
            "symbol": "crvHUSD",
            "address": "0x5B5CFE992AdAC0C9D48E05854B2d91C73a003858"
        },
        {
            "name": "Curve Aave Pool yVault",
            "symbol": "yvCurve-Aave",
            "address": "0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab"
        },
        {
            "name": "Curve Aave Pool",
            "symbol": "crvAAVE",
            "address": "0xFd2a8fA60Abd58Efe3EeE34dd494cD491dC14900"
        },
        {
            "name": "Curve ankrETH Pool yVault",
            "symbol": "yvCurve-ankrETH",
            "address": "0xE625F5923303f1CE7A43ACFEFd11fd12f30DbcA4"
        },
        {
            "name": "Curve ankrETH Pool",
            "symbol": "crvANKR",
            "address": "0xaA17A236F2bAdc98DDc0Cf999AbB47D47Fc0A6Cf"
        },
        {
            "name": "Curve sETH Pool yVault",
            "symbol": "yvCurve-sETH",
            "address": "0x986b4AFF588a109c09B50A03f42E4110E29D353F"
        },
        {
            "name": "Curve sETH Pool",
            "symbol": "crvSETH",
            "address": "0xA3D87FffcE63B53E0d54fAa1cc983B7eB0b74A9c"
        },
        {
            "name": "Curve sAave Pool yVault",
            "symbol": "yvCurve-sAave",
            "address": "0xBacB69571323575C6a5A3b4F9EEde1DC7D31FBc1"
        },
        {
            "name": "Curve sAave Pool",
            "symbol": "crvSAAVE",
            "address": "0x02d341CcB60fAaf662bC0554d13778015d1b285C"
        },
        {
            "name": "Curve Iron Bank Pool",
            "symbol": "crvIB",
            "address": "0x5282a4eF67D9C33135340fB3289cc1711c13638C"
        },
        {
            "name": "Curve Iron Bank Pool yVault",
            "symbol": "yvCurve-IronBank",
            "address": "0x27b7b1ad7288079A66d12350c828D3C00A6F07d7"
        },
        {
            "name": "Curve USDP Pool",
            "symbol": "crvUSDP",
            "address": "0x7Eb40E450b9655f4B3cC4259BCC731c63ff55ae6"
        },
        {
            "name": "Curve USDP Pool yVault",
            "symbol": "yvCurve-USDP",
            "address": "0x1B5eb1173D2Bf770e50F10410C9a96F7a8eB6e75"
        },
        {
            "name": "WBTC yVault",
            "symbol": "yvWBTC",
            "address": "0xcB550A6D4C8e3517A939BC79d0c7093eb7cF56B5"
        },
        {
            "name": "Wrapped BTC",
            "symbol": "WBTC",
            "address": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599"
        },
        {
            "name": "Curve LINK Pool yVault",
            "symbol": "yvCurve-LINK",
            "address": "0x96Ea6AF74Af09522fCB4c28C269C26F59a31ced6"
        },
        {
            "name": "Curve LINK Pool",
            "symbol": "crvLINK",
            "address": "0xcee60cFa923170e4f8204AE08B4fA6A3F5656F3a"
        },
        {
            "name": "1INCH yVault",
            "symbol": "yv1INCH",
            "address": "0xB8C3B7A2A618C552C23B1E4701109a9E756Bab67"
        },
        {
            "name": "1INCH Token",
            "symbol": "1INCH",
            "address": "0x111111111117dC0aa78b770fA6A738034120C302"
        }
    ]
}
export default yearnTokenlist